### PR TITLE
add description in OceanCalloutList

### PR DESCRIPTION
**Pull request by FariasBlu**
## Description

O componente OceanCallouList utiliza o OceanUnorderedListItemModel / OceanUnorderedListItem. Adicionei um campo para descrição em ambos. 

Anteriormente, o componente utilizava LazyColumn e quando combinado com uma Column, estava gerando essa exceção: 

java.lang.IllegalStateException: Vertically scrollable component was measured with an infinity maximum height constraints, which is disallowed.

Por este motivo, alterei de LazyColumn para Column:

image

image

Fixes # (issue)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Checklist:

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) document
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have made corresponding changes to the documentation (if appropriate)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] All new and existing tests passed

### CODE REVIEW

File: ocean-components/src/main/java/br/com/useblu/oceands/components/compose/list/OceanUnorderedList.kt [hunk 1]
```diff
@@ -112,9 +116,18 @@ fun OceanUnorderedListItem(
...
-        )
+        Column(
+            modifier = Modifier
+                .padding(start = 8.dp)
+        ) {
+            OceanText(
+                text = title,
+                style = style
+            )
+            OceanText(
```
Comment (fernandozanutto) [inline ocean-components/src/main/java/br/com/useblu/oceands/components/compose/list/OceanUnorderedList.kt:127]: coloca um if aqui pra ver se a description nao é uma string vazia, pq acho que ele ta renderizando o OceanText e ocupando espaço ali

Comment (FariasBlu) [reply ocean-components/src/main/java/br/com/useblu/oceands/components/compose/list/OceanUnorderedList.kt:127]: image

Comment (fernandozanutto) [reply ocean-components/src/main/java/br/com/useblu/oceands/components/compose/list/OceanUnorderedList.kt:127]: dai da pra atualizar os prints na PR tbm

Comment (FariasBlu) [reply ocean-components/src/main/java/br/com/useblu/oceands/components/compose/list/OceanUnorderedList.kt:127]: Opa, essa não pode faltar, tinha atualizado ali.

image

**Review by fernandozanutto — APPROVED** [review]

**Pull request merged**